.github/issue_template/request-an-update.md

```diff
@@ -0,0 +1,19 @@
+---
+name: Request an Update
```

Comment:
There's a lot of overlap with the template for new features. We may want to make this one more generalized?